### PR TITLE
Add hooks for sars genome

### PR DESCRIPTION
**Pull request by nathandunn**
- Fixes issues with yeast
- Handles SARS data (hopefully).   Had a weird proxy issue, but it should work.  There is a non-default track, so added to constants.

### CODE REVIEW

**Comment by nathandunn** [comment]
FYI @chris-grove  / @scottcain

File: src/constants.js [hunk 1]
```diff
@@ -229,12 +236,16 @@ export const SPECIES = [
...
     apolloName: 'SARS-CoV-2',
+    apolloTrack: '/Mature%20peptides%20(RefSeq)/',
+    suppressFlatten: true,
     vertebrate: false,
     enableSingleCellExpressionAtlasLink: false,
     enableOrthologComparison: false,
   }
 ];
 
+export const findSpeciesForTaxon = (taxonId) => SPECIES.find( s => (s.taxonId === taxonId));
```
Comment (paaatrick) [inline src/constants.js:247]: This function already exists as `getSpecies` in `src/lib/utils.js`

**Comment by nathandunn** [comment]
Okay @paaatrick  it is ready for review

**Comment by paaatrick** [comment]
Can you remove that redundant function?

**Comment by nathandunn** [comment]
’Tis removed.

**Comment by paaatrick** [comment]
Thanks

**Pull request merged**